libraries/Wire/README.md

```diff
@@ -0,0 +1,19 @@
+# Wire
+
+The STM32 Wire library is fully compatible with the Arduino Wire specifications.
+
+## API note
+
+Two new method has been added to allow the use of large number of bytes to write
+or read.
+
+* **writeBuffer(uint8_t \* pdata, size_t quantity)**: pass pdata directly to the
+I2C layer withtout fill the internal Wire library buffer. This method must be
+called only once after _beginTransmission()_. _endTransmission()_ must be called
+immediately after. No other _write()_ method have to be called between
+_beginTransmission()_ and _endTransmission()_ when _writeBuffer()_ is used.
+
```

Comment:
I understand the idea and this would work ok - but I am not sure that this requirement (call only once, call endTRansmission immediately after ...) is acceptable - Isn't there an alternative that is more compliant with arduino defaut API ? Coudln't we simply increase internal buffer size ?  Or maybe manage malloc & free of extra buffers in case size becomes > 255  then in endTransmission you would send the default buffers + extra ones ? ... other feedbacks are welcome of course

Reply:
We have discussed with @fprwi6labs  about that. Changing the static buffer to a dynamic one will consume memory space.  If several I2C instances are used and extra buffer size is required this will reach quickly the maximum space.
Proposal was to kept the current write function and add a new one with the usage restriction.